src/content/components/TreeView.js

```diff
@@ -367,6 +402,11 @@ class TreeView extends Component {
 
   render() {
     const { fixedColumns, mainColumn, disableOverscan } = this.props;
+    let specialItems = this._specialItems;
+    if (this._forceRender.length) {
+      specialItems = [...specialItems, ...this._forceRender];
+      this._forceRender = [];
+    }
```

Comment:
I couldn't find a better way, I know this is ugly :/

The issue is that we want `specialItems` to stay accross renders, but not `forceRender`, from what I suppose.